tests/PgnTest.php

```diff
@@ -148,6 +144,11 @@ public function testValidatePgn(): void
         $this->assertTrue($parsed);
 
         $parsed = Validation::validatePgn('1.e4 e5 2.Nf3', ['verbose' => true]);
+        if (!\is_array($parsed)) {
+            $this->markTestSkipped();
```

Comment:
Why skipped, shouldn't be failed?

Reply:
Well, I guess it's the same. It's a case limit and actually it should never be something different from an array

Reply:
then it is better to throw an exception, in some cases skipped can be treated as ok :wink: